cognite/client/experimental/model_hosting/schedules.py

```diff
@@ -22,6 +22,33 @@ class ScheduleCollectionResponse(CogniteCollectionResponse):
     _RESPONSE_CLASS = ScheduleResponse
 
 
+class LogEntry(CogniteResponse):
+    def __init__(self, internal_representation):
+        super().__init__(internal_representation)
+        item = self.to_json()
+        print("hey hey")
+        print(item)
```

Comment:
remove these print statements

Reply:
Wow, that was sloppy...

Reply:
I do it all the time. We should actually add a check in Jenkins which scans the library for print()s.